spec/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ ] It will be great to ensure that you test all the custom methods. In your user model, there is a custom method for returning `recent_posts`. But in your test cases, this method is not tested. Kindly write a unit test for this method

Reply:
Hi @peterrobert 
I think this method is tested in user_spec.rb line 22. 
the method only checks the last 3 posts.

Reply:
Hi @brainskev, this might have been missed by the previous reviewer I see the change requested is currently implemented, I will approve this review, all the best 👍🏿